src/librustc_codegen_llvm/lib.rs

```diff
@@ -270,6 +272,47 @@ impl CodegenBackend for LlvmCodegenBackend {
         )
     }
 
+    fn link(
+        &self,
+        sess: &Session,
+        input: &Input,
```

Comment:
```
        codegen_results: CodegenResults,
```

and then move all code to read the `.rlink` file to `rustc_driver`  That code is the same for all codegen backends.

Reply:
Will do once #68601 is merged.

Reply:
Done.